cypress/support/step-definitions/select-steps.js

```diff
@@ -84,11 +63,11 @@ When("{string} option on the list is highlighted", (position) => {
 });
 
 When("I click onto controlled select using {string} key", (key) => {
-  simpleSelectID().trigger("keydown", keyCode(key));
+  commonDataElementInputPreview().trigger("keydown", keyCode(key));
 });
 
-When("I click onto default select using {string} key", (key) => {
-  simpleSelect().trigger("keydown", keyCode(key));
+When("I press the {string} key, when focused on the input", (key) => {
+  selectInput().trigger("keydown", { ...keyCode(key), force: true });
```

Comment:
the same question about `force` here :)

Reply:
to hide the caret and still be ARIA compatible